test/replicatedmap/ReplicatedMapProxyTest.js

```diff
@@ -0,0 +1,130 @@
+var expect = require('chai').expect;
+var HazelcastClient = require('../../lib/index.js').Client;
+var Controller = require('./../RC');
+var Util = require('./../Util');
+var fs = require('fs');
+
+var Promise = require('bluebird');
+
+describe('ReplicatedMap Proxy', function () {
```

Comment:
In tests it is better if you do not use nested functions but return promises from each function. For example in `puts one entry and gets one entry` test, instead of 
```javascript
    it('should contain the key', function () {
        return rm.put('key', 'value', ONE_HOUR)
            .then(function (val) {
                return rm.containsKey('key')
                    .then(function (res) {
                        expect(res).to.equal(true);
                    });
            });
    });
```
you can write
```javascript
    it('should contain the key', function () {
        return rm.put('key', 'value', ONE_HOUR).then(function (val) {
            return rm.containsKey('key');
        }).then(function (res) {
            return expect(res).to.equal(true);
        });
    });
```